server_admin/topics/clients/oidc/con-secret-rotation.adoc

```diff
@@ -0,0 +1,28 @@
+[id="con-secret-rotation_{context}"]
+[[_secret_rotation]]
+= Client Secret Rotation
+[role="_abstract"]
+For client with <<_client-credentials, Confidential>> <<_access-type, Access Type>> {project_name} supports the functionality of rotating client secrets through <<_client_policies, Client Policies>>.
```

Comment:
For a client with <<_client-credentials, Confidential>> <<_access-type, Access Type>>, {project_name} supports the functionality of rotating client secrets through <<_client_policies, Client Policies>>.